internal/fnruntime/runner.go

```diff
@@ -393,8 +395,7 @@ func populateResourceRef(item *yaml.RNode, resultItem *framework.Result) error {
 
 // printFnResult prints given function result in a user friendly
 // format on kpt CLI.
-func printFnResult(ctx context.Context, fnResult *fnresult.Result, opt *printer.Options) {
-	pr := printer.FromContextOrDie(ctx)
+func printFnResult(fnResult *fnresult.Result) string {
```

Comment:
nit: If we are taking the printer out, may be the name of the function should also be changed to `fnResultToStr` or something.